Index: src/main/java/org/example/configuratoreauto/Utenti/Cliente.java
===================================================================
diff --git a/src/main/java/org/example/configuratoreauto/Utenti/Cliente.java b/src/main/java/org/example/configuratoreauto/Utenti/Cliente.java
deleted file mode 100644
--- a/src/main/java/org/example/configuratoreauto/Utenti/Cliente.java	(revision b6b90ee341bec2983dcd58de6513460d36ec7b66)
+++ /dev/null	(revision b6b90ee341bec2983dcd58de6513460d36ec7b66)
@@ -1,9 +0,0 @@
-package org.example.configuratoreauto.Utenti;
-
-public class Cliente extends Persona{
-
-    public Cliente(String email, String password){
-        super(email, password);
-        super.userType = TipoUtente.cliente;
-    }
-}
